Library/.rubocop.yml

```diff
@@ -197,6 +197,15 @@ Layout/RescueEnsureAlignment:
 Lint/AmbiguousBlockAssociation:
   Enabled: false
 
+# needed for RSpec tests and stubbing in formulae
+Lint/EmptyBlock:
+  Enabled: false
```

Comment:
Can we exclude the specific files instead or provide some examples? I'd like to get these things fixed up.